src/main/java/com/pnu/dev/radioserviceapi/service/BannerApiService.java

```diff
@@ -0,0 +1,12 @@
+package com.pnu.dev.radioserviceapi.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+
+@Service
```

Comment:
No needs in this annotation here

Reply:
Thank you for being so attentive, I missed it.